src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -909,7 +909,6 @@ private MenuBar createMenu() {
                 factory.createMenuItem(StandardActions.SETUP_GENERAL_FIELDS, new SetupGeneralFieldsAction()),
                 factory.createMenuItem(StandardActions.MANAGE_CUSTOM_IMPORTS, new ManageCustomImportsAction()),
                 factory.createMenuItem(StandardActions.MANAGE_CUSTOM_EXPORTS, new ManageCustomExportsAction()),
-                factory.createMenuItem(StandardActions.MANAGE_EXTERNAL_FILETYPES, new EditExternalFileTypesAction()),
```

Comment:
Please check the GitHub actions Checkstyle output:

    Error: eckstyle] [ERROR] /home/runner/work/jabref/jabref/src/main/java/org/jabref/gui/JabRefFrame.java:84:8: Unused import - org.jabref.gui.externalfiletype.EditExternalFileTypesAction. [UnusedImports]